arki55/fuse-fuse
# fix: Fixed handling of multiple monitors [win32 ui]  ==>  master

Testing protocol for this bug (only win32 UI):
- Open Fuse, move window around
- selecting different machine on primary screen
- select different machine on secondary screen
- minimize and resume window on primary screen
- minimize and resume window on secondary screen
- change size of window on primary screen - should allow sizes possible on this screen
- change size of window on secondary screen - should allow sizes possible on this screen

See also
https://sourceforge.net/p/arki55-fuse-mod/tickets/8/
